Public/Src/Cache/DistributedCache.Host/Service/Internal/DistributedContentStoreFactory.cs

```diff
@@ -451,5 +448,43 @@ public KeyVaultConfigurationException(string message, Exception innerException)
             {
             }
         }
+
+        private static RetryPolicy CreateKeyVaultRetryPolicy()
+        {
+            return new RetryPolicy(
+                new KeyVaultRetryPolicy(),
+                new ExponentialBackoff(
+                    name: "KeyVaultExponentialBackoff",
+                    retryCount: 5,

```

Comment:
>5 [](start = 32, length = 1)

Should this, minimum backoff, maximum backoff, and delta backoff be configurable?

Reply:
It may, but this is a temporary solution before we move to the secret store. And we're not planning to use key vault in a way that will cause issues even without the backoff algorithm. So I'm not sure the configuration is worth it.